src/hooks/useDydxClient.tsx

```diff
@@ -169,14 +170,17 @@ const useDydxClientContext = () => {
     }
   };
 
-  const getMegavaultHistoricalPnl = useCallback(async () => {
-    try {
-      return await indexerClient.vault.getMegavaultHistoricalPnl();
-    } catch (error) {
-      log('useDydxClient/getMegavaultHistoricalPnl', error);
-      return undefined;
-    }
-  }, [indexerClient.vault]);
+  const getMegavaultHistoricalPnl = useCallback(
+    async (resolution: PnlTickInterval = PnlTickInterval.day) => {
+      try {
+        return await indexerClient.vault.getMegavaultHistoricalPnl(resolution);
```

Comment:
why not just wrap this with `  VaultCalculator.getVaultHistoricalPnlResponse(safeStringifyForAbacusParsing(res));` here